packages/video_player/video_player/ios/Classes/messages.m

```diff
@@ -0,0 +1,289 @@
+// Autogenerated from Pigeon (v0.1.0-experimental.10), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+#import "messages.h"
+#import <Flutter/Flutter.h>
+
+#if !__has_feature(objc_arc)
+#error File requires ARC to be enabled.
+#endif
+
+static NSDictionary *wrapResult(NSDictionary *result, FlutterError *error) {
+  NSDictionary *errorDict = (NSDictionary *)[NSNull null];
+  if (error) {
+    errorDict = [NSDictionary
+        dictionaryWithObjectsAndKeys:(error.code ? error.code : [NSNull null]), @"code",
+                                     (error.message ? error.message : [NSNull null]), @"message",
+                                     (error.details ? error.details : [NSNull null]), @"details",
+                                     nil];
+  }
+  return [NSDictionary dictionaryWithObjectsAndKeys:(result ? result : [NSNull null]), @"result",
+                                                    errorDict, @"error", nil];
+}
+
+@interface FLTTextureMessage ()
++ (FLTTextureMessage *)fromMap:(NSDictionary *)dict;
+- (NSDictionary *)toMap;
+@end
+@interface FLTCreateMessage ()
++ (FLTCreateMessage *)fromMap:(NSDictionary *)dict;
+- (NSDictionary *)toMap;
+@end
+@interface FLTLoopingMessage ()
++ (FLTLoopingMessage *)fromMap:(NSDictionary *)dict;
+- (NSDictionary *)toMap;
+@end
+@interface FLTVolumeMessage ()
++ (FLTVolumeMessage *)fromMap:(NSDictionary *)dict;
+- (NSDictionary *)toMap;
+@end
+@interface FLTPositionMessage ()
++ (FLTPositionMessage *)fromMap:(NSDictionary *)dict;
+- (NSDictionary *)toMap;
+@end
+
+@implementation FLTTextureMessage
++ (FLTTextureMessage *)fromMap:(NSDictionary *)dict {
+  FLTTextureMessage *result = [[FLTTextureMessage alloc] init];
+  result.textureId = dict[@"textureId"];
+  if ((NSNull *)result.textureId == [NSNull null]) {
+    result.textureId = nil;
+  }
+  return result;
+}
+- (NSDictionary *)toMap {
+  return
+      [NSDictionary dictionaryWithObjectsAndKeys:(self.textureId ? self.textureId : [NSNull null]),
```

Comment:
Good call.  I would update this but since it is part of the generated code it is really an issue with pigeon, not this PR.  That requires republishing pigeon, since it's a nit I'll address it later if you don't mind.